### PR TITLE
manually call delete on GLCanvas in Application dtor

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by birarda** [comment]
This fixes a crash on shutdown on Windows - the GLCanvas timer was still alive and was calling throttleRender which called paintGL which needs the TextureCache which had already been destroyed by the Application destructor

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -629,7 +631,7 @@ void Application::initializeGL() {
     #ifdef WIN32
     GLenum err = glewInit();
     if (GLEW_OK != err) {
-      /* Problem: glewInit failed, something is seriously wrong. */
+      /* Problem: glewInit fa   iled, something is seriously wrong. */
```
Comment (jherico) [inline interface/src/Application.cpp]: typo?

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -602,7 +602,9 @@ Application::~Application() {
 
     _myAvatar = NULL;
 
-    ModelEntityItem::cleanupLoadedAnimations() ;
+    ModelEntityItem::cleanupLoadedAnimations();
+    
+    delete _glWidget;
```
Comment (jherico) [inline interface/src/Application.cpp]: should we not also set it to nullptr?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 292354e180a0d37926fcf68419ac5ce80b0f906d available at: https://gist.github.com/808b3b71edc7e5c90cb0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4405.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4405.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a8046b8316bf477073fa37910fdf942f43146fcb available at: https://gist.github.com/808b3b71edc7e5c90cb0<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4405.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4405.dmg

**Pull request merged**